README.md

```diff
@@ -66,6 +66,11 @@ MQTTnet is a high performance .NET library for MQTT based communication. It prov
 | Xamarin.iOS        | 10.14+      |
 | Blazor WebAssembly | 3.2.0+      |
 
+## Build requirements
+
+* Visual Studio 2022 (Net6.0 is targeted in the project and Visual Studio 2019 won't work with Net6.0)
+* UWP Support for Visual Studio (Windows SDK for UWP version 10.0.18362.0+)
```

Comment:
We need exactly 10.0.18362.0. This value is also configured in the .csproj file. There is also the minimum compatible version specified.

Reply:
Ok, thanks for the information. I have no idea since I only use NetFramework 4.8 (Where I didn't have the energy to solve Net6.0 compatibility issues) and Net6.0 for now 😄